Blackjack/Controller.cs

```diff
@@ -68,14 +70,29 @@ public void Play()
                 }
                 // parse input
                 choice = ChoiceParser.ParseChoice(input);
-                if (choice == Choice.Hit) DealCard(player);
+                if (choice == Choice.Hit) 
+                {
+                    var playerCard = DealCard(player);
+                    _output.WriteLine(String.Format(Messages.PlayerCard, OutputFormatter.DisplayCard(playerCard))); // TODO: in test card is null
+                }
                 // calculate player's score
-                // var score = Score.Calculate(Players[0].Hand); 
-                // TODO: how to add the players' hand. 'Players[0].Hand' - error = cannot convert from 'Blackjack.IHand' to 'Blackjack.Hand'
+                playerScore = Score.Calculate(player.Hand);
+                _output.WriteLine(String.Format(Messages.PlayerScore, playerScore, OutputFormatter.DisplayHand(player.Hand)));
             }
             while (choice != Choice.Stay);

```

Comment:
What other conditions could end the game here for the player? i.e. check if `Blackjack` or `Bust`